docs/architecture/architecture.md

```diff
@@ -609,6 +614,133 @@ rootfs by calling `mount(8)` within the counter.  If the devicemapper block devi
 is used, `/` will be mounted on `/dev/vda`.  Users can disable direct mounting
 of the underlying block device through the runtime configuration.
 
+## Kubernetes support
+[Kubernetes\*](https://github.com/kubernetes/kubernetes/) is a popular open source
+container orchestration engine. It runs a control plane where a scheduler
+(typically running on a dedicated master node) calls into a compute node container
+agent called `kubelet`. On each compute node a `kubelet` instance is responsible
+for managing the node pods lifecycle and eventually relies on a container runtime for
+handling the container execution parts. The `kubelet` architecture decouples the
+lifecycle management part from the container execution one through the dedicated
+[`gRPC`](https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/apis/cri/v1alpha1/runtime/api.proto) 
+[Container Runtime Interface (CRI)](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/node/container-runtime-interface-v1.md).
+In other words, `kubelet` is a `CRI` client and expects a `CRI` implementation to
+handle the server side of the interface.
+[CRI-O\*](https://github.com/kubernetes-incubator/cri-o) is a `CRI` implementation
+that relies on `OCI` compatible runtimes for managing container instances.
+
+Clear Containers, being an OCI compatible runtime, naturally fits into the `CRI-O`
+architecture and requirements, and is one of the two officially supported `CRI-O` runtimes.
```

Comment:
Note that by stating "two" we will have to maintain this document and update the number.
To avoid this:

Clear Containers is an officially supported CRI-O runtime. It is OCI compatible and therefore aligns with CRI-Oarchitecture and requirements.